packages/vm/tests/runner.ts

```diff
@@ -0,0 +1,63 @@
+const level = require('level')
+import { Transaction } from '@ethereumjs/tx'
+import VM from '../lib'
+import { SecureTrie as Trie } from '@ethereumjs/trie'
+import { DefaultStateManager } from '../lib/state'
+import Common from '@ethereumjs/common'
+import { Block } from '@ethereumjs/block'
+
+const main = async () => {
+  const txData = {
+    nonce: '0x1a',
+    gasPrice: '0xba43b7400',
+    gasLimit: '0x493e0',
+    to: '0x9ca228250f9d8f86c23690074c2b96d5f5479f79',
+    value: '0x0',
+    data:
+      '0xa9059cbb000000000000000000000000bfe465e7eb5a2928b5bf22bef93ad06089dc6179000000000000000000000000000000000000000000000000006a94d74f430000',
+    v: '0x1c',
+    r: '0xc04f048766bea3b20dcea6d2fcaecc92f1a615d337606ab6eec2a1e8e1f27bfe',
+    s: '0x1c57e62653724560b06c95be750d958840db9fa44ffa777b6625b344bfadbbcf',
+  }
+  const common = new Common({ chain: 'mainnet', hardfork: 'chainstart' })
+  const tx = Transaction.fromTxData(txData, { common })
+  const trie = new Trie(level('/Users/jochem/Library/Ethereum/ethereumjs/mainnet/state/'))
+
+  // Ensure we run on the right root
+  trie._setRoot(
+    Buffer.from('9dceb42e227ee6a244449d1d92cc5e0c17c63b520d0ff050d943baa8055d0ca6', 'hex')
+  )
```

Comment:
Actually the more official way is doing a `stateManager.setStateRoot(Buffer.from('9dceb42e227ee6a244449d1d92cc5e0c17c63b520d0ff050d943baa8055d0ca6', 'hex'))` here. 😄

Reply:
Ah yes, I realized later on that I should not call methods on the low-level trie, but indeed on the state manager. (This PR is rather dirty ATM - it is a sort of PoC)